src/com/nutrons/framework/controllers/Events.java

```diff
@@ -0,0 +1,53 @@
+package com.nutrons.framework.controllers;
+
+public class Events {
```

Comment:
@kreeve I don't think we want to mock these events.  It's probably better to use equality on events in tests and have special mocked versions of the producer and consumer classes instead.